### PR TITLE
288 jump to nucleotide

### PR DESCRIPTION
**Pull request by aminere**
Use hamburger menu -> `Jump to Nucleotide` or press `J` to open the nucleotide finder.

- The nucleotide is index is 1-based. If custom numbering is present on a pose, it's used to identify the nucleotide index.
- Handles PiP mode by focusing the same nucleotide in all poses (thanks Jonathan)
- The new icon in the hamburger menu is a repurposed zoom icon, we can replace it.

### CODE REVIEW

File: src/eterna/ui/Toolbar.ts [hunk 1]
```diff
@@ -279,6 +280,16 @@ export default class Toolbar extends ContainerObject {
             this.actionMenu.addSubMenuButton(0, this.pasteButton);
         }
 
+        this.nucleotideFindButton = new GameButton()
+            .allStates(Bitmaps.ImgFind)
+            .disabled(null)
+            .label('Jump to Nucleotide', 14)
+            .scaleBitmapToLabel()
+            .tooltip('Type a nucleotide index to jump to it')
```
Comment (luxaritas) [inline src/eterna/ui/Toolbar.ts:288]:
```suggestion
            .tooltip('Type a nucleotide index to put it in the center of the screen (j)')
```

File: src/eterna/mode/PoseEdit/PoseEditMode.ts [hunk 2]
```diff
@@ -158,6 +159,24 @@ export default class PoseEditMode extends GameMode {
         this._toolbar.pairSwapButton.clicked.connect(() => this.onSwapClicked());
         this._toolbar.hintButton.clicked.connect(() => this.onHintClicked());
 
+        this._toolbar.nucleotideFindButton.clicked.connect(() => {
+            if (this._nucleotideFinderRef.isLive) {
+                return;
+            }
+
+            const {panel, positionUpdater} = NucleotideFinder.create({
```
Comment (luxaritas) [inline src/eterna/mode/PoseEdit/PoseEditMode.ts:167]: Should this be handled something like how this is instead? https://github.com/eternagame/EternaJS/blob/master/src/eterna/mode/PoseEdit/SubmitPoseDialog.ts

Comment (aminere) [reply src/eterna/mode/PoseEdit/PoseEditMode.ts]: Indeed! Works great

File: src/eterna/ui/Toolbar.ts (hunk 1 above)
Comment (aminere) [reply src/eterna/ui/Toolbar.ts]: Updated

**Pull request merged**